components/PriceFilter.tsx

```diff
@@ -1,10 +1,10 @@
 import { useState } from 'react';
 
 interface PriceFilterProps {
-  handleApplyClick: (min: number, max: number) => void;
+  handleApplyPriceFilterClick: (min: number, max: number) => void;
```

Comment:
컴포넌트가 props로 이벤트 핸들러 함수를 받을때는 on- prefix를 보통 사용하는 편이기도 해요~ 팀 컨벤션에 따라가면 될 것 같아요.

```suggestion
  onApplyPriceFilterClick: (min: number, max: number) => void;
```

Reply:
PriceFIlter 컴포넌트에서 onChange에 인라인 함수를 사용하지 말고 상위에서 함수를 만들어서 써주면 좋을 것 같아요.
`handleMinChange`, `handleMaxChange` 이렇게요~

```tsx
        <input
          type='number'
          placeholder='최소'
          value={min}
          onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
            setMin(e.target.valueAsNumber);
          }}
        />
        <input
          type='number'
          placeholder='최대'
          value={max}
          onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
            setMax(e.target.valueAsNumber);
          }}
        />
```